addons/payment/models/res_company.py

```diff
@@ -55,7 +55,16 @@ def _run_payment_onboarding_step(self, menu_id):
             [('type', '=', 'bank'), ('company_id', '=', new_env.company.id)], limit=1
         )
 
-        stripe_acquirer = new_env.ref('payment.payment_acquirer_stripe')
+        stripe_counter = new_env['payment.acquirer'].search_count(
+            [('company_id', '=', self.env.company.id), ('name', '=', 'Stripe')]
+        )
+        if stripe_counter == 0:
+            new_acquirer = self.env.ref('payment.payment_acquirer_stripe').with_company(self.env.company.id).sudo().copy()
+            new_acquirer.company_id = self.env.company.id
+        stripe_acquirer = new_env['payment.acquirer'].search(
+            [('company_id', '=', self.env.company.id), ('name', '=', 'Stripe')],
+            limit=1
+        )
```

Comment:
Why a counter is needed here?
```suggestion
        stripe_acquirer = new_env['payment.acquirer'].search(
            [('company_id', '=', self.env.company.id), ('name', '=', 'Stripe')],
            limit=1
        )
        if not stripe_acquirer:
            stripe_acquirer = self.env.ref('payment.payment_acquirer_stripe').with_company(self.env.company.id).sudo().copy()
            stripe_acquirer.company_id = self.env.company.id
```